Azure/azure-ad/ingest/parser.yml

```
@@ -156,6 +164,17 @@ stages:
           azuread.callerIpAddress: "{{parsed_event.message.callerIpAddress}}"
         filter: "{{parsed_event.message.callerIpAddress | is_ipaddress}}"
 
+      - set:
+          user_agent.original: >
+            {%- for item in parsed_additional_info.message -%}
+              {%- if item.Key == 'userAgent' -%}{{ item.Value }}{%- endif -%}
+            {%- endfor -%}
+
+          azuread.riskReasons: >
+            {%- for item in parsed_additional_info.message -%}
+              {%- if item.Key == 'riskReasons' -%}{{ item.Value }}{%- endif -%}
+            {%- endfor -%}
+
```

Can you move this field to `azuread.properties.riskReasons`?
```suggestion
          azuread.properties.riskReasons: >
            {%- for item in parsed_additional_info.message -%}
              {%- if item.Key == 'riskReasons' -%}{{ item.Value }}{%- endif -%}
            {%- endfor -%}

```